haystack/api/controller/model.py

```diff
@@ -119,7 +120,8 @@ def doc_qa(model_id: int, request: Question):
             )
             results.append(result)
 
-            logger.info({"request": request.json(), "results": results})
+        elasticapm.set_custom_context({"results": results})
```

Comment:
What if there's no APM server configured? I think we need an `if APM_SERVER:` block here, right?

Reply:
It should be fine. I think APM won't raise an error if not configured.